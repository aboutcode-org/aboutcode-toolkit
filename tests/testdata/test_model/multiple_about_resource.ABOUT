about_resource:
 - .
 - multiple_files_expected.csv
name: multiple_about_resource
