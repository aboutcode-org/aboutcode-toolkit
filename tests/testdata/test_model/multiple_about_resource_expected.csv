about_resource,name
"/test_model/
/test_model/multiple_files_expected.csv",multiple_about_resource
